plugins/webp-uploads/hooks.php

```diff
@@ -534,22 +534,26 @@ function webp_uploads_update_image_references( $content ): string {
 
 	// This content does not have any tag on it, move forward.
 	// TODO: Eventually this should use the HTML API to parse out the image tags and then update them.
-	if ( ! preg_match_all( '/<(img)\s[^>]+>/', $content, $img_tags, PREG_SET_ORDER ) ) {
+	if ( 0 === (int) preg_match_all( '/<(img)\s[^>]+>/', $content, $img_tags, PREG_SET_ORDER ) ) {
 		return $content;
 	}
 
 	$images = array();
 	foreach ( $img_tags as list( $img ) ) {
 		$processor = new WP_HTML_Tag_Processor( $img );
-		if ( ! $processor->next_tag( array( 'tag_name' => 'IMG' ) ) ) {
+		if ( 0 === (int) $processor->next_tag( array( 'tag_name' => 'IMG' ) ) ) {
```

Comment:
This change was not warranted.

```suggestion
		if ( ! $processor->next_tag( array( 'tag_name' => 'IMG' ) ) ) {
```